### PR TITLE
debugging ci faliures

### PR DESCRIPTION
**Pull request by Cadair**
@astrofrog

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -101,6 +101,7 @@ matrix:
 
 install:
     - git clone git://github.com/astropy/ci-helpers.git
+    - cat ci-helpers/travis/setup_dependencies_common.sh
```
Comment (dpshelio) [inline .travis.yml]: :smile_cat:

**Comment by ayshih** [comment]
These are some interesting commit messages that are flying across my radar...

**Comment by Cadair** [comment]
@ayshih Debugging CI: the poster child for squash rebases!

**Comment by Cadair** [comment]
Review please!

@sunpy/sunpy-developers

File: .travis.yml [hunk 2]
```diff
@@ -21,83 +16,56 @@ addons:
...
-#    directories:
-#       - $HOME/miniconda/pkgs
-#       - $HOME/miniconda3/pkgs
-
 # Configure the build environment. Global varibles are defined for all configurations.
 env:
     global:
         # Modify these to represent the newest versions
         - PREVIOUS_PANDAS=0.15.2
         - PREVIOUS_NUMPY=1.9.3
```
Comment (bsipocz) [inline .travis.yml]: this is not true as current numpy is 1.11

**Comment by bsipocz** [comment]
looks like you dont have a build when only hard dependencies are install but none of the optional ones.

**Comment by Cadair** [comment]
@bsipocz I will put a no optional dependancies build on the todo list.

**Pull request merged**